src/NHapi.Base/SupportClass.cs

```diff
@@ -2130,6 +2131,19 @@ public static object Pop(ArrayList stack)
             }
         }
 
+        /// <summary>
+        /// Evaluates if a string is comprised of unique characters.
+        /// </summary>
+        /// <param name="input">string to evaluate.</param>
+        /// <returns>True if all characters are unique, otherwise False.</returns>
+        public static bool CharsAreUnique(string input)
+        {
+            var chars = input.ToCharArray();
```

Comment:
will `.ToCharArray()` throw if input is null?

Reply:
most likely - will make some changes.